plugin/trino-base-jdbc/src/main/java/io/trino/plugin/jdbc/JdbcClient.java

```diff
@@ -184,7 +185,7 @@ default void setColumnComment(ConnectorSession session, JdbcTableHandle handle,
         throw new TrinoException(NOT_SUPPORTED, "This connector does not support setting column comments");
     }
 
-    void addColumn(ConnectorSession session, JdbcTableHandle handle, ColumnMetadata column);
+    void addColumn(ConnectorSession session, JdbcTableHandle handle, ColumnMetadata column, ColumnPosition position);
```

Comment:
The base jdbc module is used likes a library from 3rd party repositories. Can you try keeping the backward compatibility with a default method?

Reply:
In what condition we can confirm that the method can be safe remove?